### PR TITLE
Update packages

### PR DESCRIPTION
**Pull request by neilbrub**
`npm audit fix` to patch security vulnerabilities in old package versions.

-- DON'T MERGE YET --

### CODE REVIEW

**Comment by neilbrub** [comment]
TL;DR: nasty compatibility issues pose an obstacle for removing these vulnerabilities through package updates, aborting this attempt for now.

Updating vulnerable packages includes moving to a higher version of `react-native-scripts`, which as it turns out is actually deprecated in favour of `expo` (through `expo-cli`). There seem to be compatibility issues with expo SDK 27 and `react-native` versions above 0.5 (see the [statement](https://blog.expo.io/expo-sdk-v29-0-0-is-now-available-f001d77fadf)), which also contains vulnerabilities. Further, building with the updated `react-native-scripts` also failed in a plethora of ways with the basic `npm audit fix` package updates. I played around with a couple different package versions but didn't find much success in building (although expo-cli is neat). Will give another go at this in a bit, but might require updating the expo SDK we're using (current is 32, we're on 27).
